gnd/build.gradle

```diff
@@ -53,8 +53,8 @@ android {
         // Switching targetSdkVersion to 27 or 28 causes the app to hang on map drags; diagnose and
         // fix before increasing this value.
         targetSdkVersion 26
-        versionCode gitVersioner.versionCode
-        versionName gitVersioner.versionName
+        versionCode 1
+        versionName '0.0.1'
```

Comment:
Can we continue to use auto-versions here?

Reply:
I temporarily removed this as `gitVersioner` requires `.git` directory to function (which is ignored in .gitignore)

Any ideas to overcome this problem?

Reply:
One idea is to utilize the .buildenv file added in this PR to track the version count and name.

Reply:
How that work when building directly via command-line with Gradle or Android Studio? 

If it's too difficult to keep, feel free to gitVersioner for now, but let's make sure the version number is defined in at most one place.

Reply:
Done